src/domain/SipLine.js

```diff
@@ -41,6 +43,8 @@ export default class SipLine {
   links: Array<Object>;
   trunk: ?string;
   line: Endpoint;
+  hasVideo: boolean;
+  hasVideoConference: boolean;
```

Comment:
Can we have a getter for these 2 attributes ? 
Can you update the documentation accordingly ?